ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -ex
+
+#sudo apt-get install -y rocblas rccl-dev rccl-rdma-sharp-plugins
```

Comment:
Is this line still needed? And does L6 have any relation to it?

Reply:
removed. I do want to uninstall rccl to keep only one version of rccl installed.